tests/acceptance/features/apiOcm/share.feature

```diff
@@ -618,3 +618,33 @@ Feature: an user shares resources using ScienceMesh application
         }
       }
       """
+
+  @issue-10285
+  Scenario: federated user upload file to a shared folder via TUS
+    Given using server "LOCAL"
+    And user "Alice" has created a folder "FOLDER" in space "Personal"
+    And "Alice" has created the federation share invitation
+    And using server "REMOTE"
+    And "Brian" has accepted invitation
+    And using server "LOCAL"
+    And user "Alice" has sent the following resource share invitation to federated user:
+      | resource        | FOLDER                        |
+      | space           | Personal                      |
+      | sharee          | Brian                         |
+      | shareType       | user                          |
+      | permissionsRole | Editor                        |
+      | federatedServer | @federation-ocis-server:10200 |
+    And using server "REMOTE"
+#    When user "Brian" lists the shares shared with him using the Graph API
+#    Then the HTTP status code should be "200"
+    And user "Brian" has created a new TUS resource in the space "Shares" with the following headers:
```

Comment:
looks like the federated share is not accessible similar to normal (local) shares. so we might need to use different step for accessible federation shares.

- Cannot access federation shares via webdav paths. (GET, PUT, PROPFIND) doesn't work

Reply:
See: https://github.com/owncloud/ocis/issues/10536

Reply:
Can you please add examples what hat requests exactly are failing?